challenges.yml

```diff
@@ -1699,3 +1699,448 @@ extending_smart_watcher_2:
+            return {
+                showDisclaimer: false,
+            };
+        },
+        methods: {
+            async addPizzaToCart() {
+                await buyPizza();
+            }
+        },
+    };
+    </script>
+    ```
+
+    How can we make it work using Vue transitions?
+  explanation: |
+    Using Vue transitions can seem a bit complicated at first, but it's
+    really a very usefull feature once you get to know it!
+
+    Remember the classes it relies on have to be under the `:global` keyword,
+    otrherwise your transition won't work where the class names have been
+    altered!
+  features:
+    multiple_choice:
+      choices:
+        a: |
+          ```vue
+          <template>
+              <div :class="$style.component">
+                  <button @click="addPizzaToCart" />
+
+                  <transition>
+                      <p v-show="showDisclaimer">
+                          <strong>Disclaimer</strong>: Eating too much pizza can be
+                          bad for your health!
+                      </p>
+                  </transition>
+              </div>
+          </template>
+
+          <script>
+          import { buyPizza } from '@/services/buy-pizza';
+
+          export default {
+              name: 'PizzaCheckout',
+              data() {
+                  return {
+                      showDisclaimer: false,
+                  };
+              },
+              methods: {
+                  async addPizzaToCart() {
+                      await buyPizza();
+                  }
+              },
+          };
+          </script>
+
+          <style lang="scss" module>
+          .component :global {
+              .v-enter, v-leave {
+                  opacity: 0;
+              }
+          }
+          </style>
+          ```
+        b: |
+          ```vue
+          <template>
+              <div :class="$style.component">
+                  <button @click="addPizzaToCart" />
+
+                  <transition>
+                      <p v-show="showDisclaimer">
+                          <strong>Disclaimer</strong>: Eating too much pizza can be
+                          bad for your health!
+                      </p>
+                  </transition>
+              </div>
+          </template>
+
+          <script>
+          import { buyPizza } from '@/services/buy-pizza';
+
+          export default {
+              name: 'PizzaCheckout',
+              data() {
+                  return {
+                      showDisclaimer: false,
+                  };
+              },
+              methods: {
+                  async addPizzaToCart() {
+                      await buyPizza();
+                  }
+              },
+          };
+          </script>
+
+          <style lang="scss" module>
+          .component :global {
+              .v-enter-active, .v-leave-active {
+                  transition: opacity 1s;
+              }
+
+              .v-enter, v-leave {
```

Comment:
```suggestion
              .v-enter, .v-leave {
```

Reply:
I thought we needed `.v-leave-to`. Do both work? Also, we should maybe be clear this question is for Vue 2, since Vue 3 has different classes.